index.html

```diff
@@ -8,6 +8,75 @@
     <link rel="stylesheet" href="./styles.css" />
   </head>
   <body>
+    <main>
+      <div class="container">
+        <section class="primary">
+          <h1 class="page-title">Contrast Ratio Calculator</h1>
+          <div class="row">
+            <div class="input-fields">
+              <div class="form-item">
+                <label for="foreground">Foreground color</label>
+              </div>
+              <div class="input-items">
+                <div class="swatch-one"></div>
+                <input id="foreground" name="foreground" class="form-input" type="text" value="#000000" />
```

Comment:
This should be changed to `id="foreground-color"` because will be target this input in JavaScript and it should have a more descriptive name. 

Also it should be `for="foreground-color"`